.github/workflows/ci.yml

```diff
@@ -44,3 +44,21 @@ jobs:
           cd build
           cp /usr/x86_64-w64-mingw32/lib/libwinpthread-1.dll test/ || true # used only for MinGW tests, ok to fail for other cases
           ctest --output-on-failure
+  fuzzing_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Ubuntu Build Dependencies
+        run: |
+          sudo apt update
+          sudo apt install libsdl2-dev liblzo2-dev libssl-dev gnutls-dev libgcrypt-dev mingw-w64-x86-64-dev binutils-mingw-w64-x86-64 gcc-mingw-w64-x86-64 wine clang
+      - name: Build
+        env:
```

Comment:
Wouldn't it make more sense to add these to the matrix in line 8 and then add excludes? Dunno...

Reply:
The fuzzer is not a regular test as used by `ctest` 
It will run indefinitely (or until it finds a bug), instead of quickly succeeding...

Reply:
Ah I see. Can it be made to run for say 5 minutes? So that new code is fuzzed automatically. TBH, I don't know how this is handled in other projects...